File: app/src/main/java/com/avoid/playtolearn/widget/WindowTitle.java
package com.avoid.playtolearn.widget;

import android.content.Context;
import android.graphics.Color;
import android.graphics.Typeface;
import android.util.AttributeSet;

import com.avoid.playtolearn.R;

public class WindowTitle extends android.support.v7.widget.AppCompatButton {
    public WindowTitle(Context context){
        super(context);
        applyCustomShape(context);
    }

    public WindowTitle(Context context, AttributeSet attrs){
        super(context, attrs);
        applyCustomShape(context);
    }

    public WindowTitle(Context context, AttributeSet attrs, int defStyleAttrs){
        super(context, attrs, defStyleAttrs);
        applyCustomShape(context);
    }

    private void applyCustomShape(Context context){
        setBackgroundResource(R.drawable.common_window_title);
        setTypeface(Typeface.createFromAsset(context.getAssets(), "fonts/zantroke.otf"));
        setTextSize(16);
        setTextColor(Color.parseColor("#874f21"));
    }

}

File: app/src/main/java/com/ivantha/playtolearn/widget/WindowTitle.kt
package com.ivantha.playtolearn.widget

import android.content.Context
import android.graphics.Color
import android.graphics.Typeface
import android.util.AttributeSet

import com.ivantha.playtolearn.R

class WindowTitle : android.support.v7.widget.AppCompatButton {

    init {
        applyCustomShape()
    }

    constructor(context: Context) : super(context)

    constructor(context: Context, attrs: AttributeSet) : super(context, attrs)

    constructor(context: Context, attrs: AttributeSet, defStyleAttrs: Int) : super(context, attrs, defStyleAttrs)

    private fun applyCustomShape() {
        setBackgroundResource(R.drawable.common_window_title)
        typeface = Typeface.createFromAsset(context.assets, "fonts/zantroke.otf")
        textSize = 16f
        setTextColor(Color.parseColor("#874f21"))
    }

}
